panther_lights_controller/README.md

```diff
@@ -14,37 +14,173 @@ Package used to control the Husarion Panther LED panels.
+| 1   | READY             | green expanding from center to the edges                    |
+| 2   | ERROR             | red blinking twice                                          |
+| 3   | MANUAL_ACTION     | blue expanding from the center to the edges                 |
+| 4   | AUTONOMOUS_ACTION | orange expanding from center to the edges                   |
+| 5   | GOAL_ACHIEVED     | purple blinking three times                                 |
+| 6   | LOW_BATTERY       | two orange stripes moving towards the center, repeats twice |
+| 7   | CRITICAL_BATTERY  | two red stripes moving towards the center, repeats twice    |
 
+Default animations are described and loaded on the node start, directly from `config/panther_lights_animations.yaml`. Supported keys are:
+
+- `animation` [*dict*]: definition of animation. See section below for more info.
 - `id` [*int*]: ID of an animation.
-- `animation` [*dict*]: description of animation.
-- `name` *(optional)* [*string*]: name of an animation.
-- `interrupting` *(optional)* [*bool*]: if *true* animation will interrupt currently displayed animation.
+- `interrupting` [*bool*, optional]: if *true* animation will interrupt currently displayed animation.
+- `name` [*string*, optional]: name of an animation.
 
 ### ImageAnimation
 
-Animation returning frame to display based on an image. The display duration which is a product of a single image duration and repeat count can't exceed 10 seconds. Supported keys are:
+Animation returning frame to display based on an image. Supported keys are:
 
+- `brightness` [*float*, optional]: animation brightness. This will overwrite `global_brightness` for a given animation.
+- `color` [*int*, optional]: image will be turned into grayscale and then the color will be applied with brightness from grayscale. Values have to be in HEX format.
+- `duration` [*float*]: duration of a single image animation.
+- `image` [*string*]: path to an image file. Only global paths are valid. Allows using `$(find ros_package)` syntax.
+- `repeat` [*int*, optional]: number of times the animation will be repeated, by default animation will run once.
 - `type` [*string*]: required field specyfying animation type, for `ImageAnimation` value should be `image_animation`.
-- `image` [*string*]: path to an image file, if a path is not global will look in the `animations` folder for the given name.
-- `duration` [*float*]: duration of an animation.
-- `repeat` *(optional)* [*int*]: number of times the animtion will be repeated, by default animation will run once.
-- `brightness` *(optional)* [float]: animation brightness, this will be used instead of controller `global_brightness`.
-- `color` *(optional)* [*int*]: if set the image color will be changed to the specified one, advised way is to use HEX format eg. `0xff0000`.
 
-### Defining new animation
+**NOTE:** The overall display duration of an animation is a product of a single image duration and repeat count. It can't exceed 10 seconds.
```

Comment:
```suggestion
:bulb: **NOTE:** The overall display duration of an animation is a product of a single image duration and repeat count. It can't exceed 10 seconds.
```
It might be added, might be omitted